mifosng-android/src/main/java/com/mifos/api/datamanager/DataManagerDocument.java

```diff
@@ -105,9 +105,9 @@ public DataManagerDocument(BaseApiManager baseApiManager) {
      * @param name       Name of Document
      * @param desc       Description of Document
      * @param file       Document File
-     * @return GenericResponse
+     * @return DocumentRelatedResponse
      */
-    public Observable<GenericResponse> updateDocument(String entityType, int entityId, int
+    public Observable<DocumentRelatedResponse> updateDocument(String entityType, int entityId, int
```

Comment:
@tarun0 When we know what will be response for many requests the It's better to follow genericity and Please use `GenericResponse`